File: app/src/at-dinhtruong/java/asiantech/internship/summer/recyclerview/RecyclerViewActivity.java
package asiantech.internship.summer.recyclerview;

import android.support.v4.app.FragmentTransaction;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

import asiantech.internship.summer.R;

public class RecyclerViewActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_recycler_view);
        initFragment();
    }

    private void initFragment() {
        FragmentTransaction fragmentTransaction = getSupportFragmentManager().beginTransaction();
        fragmentTransaction.replace(R.id.fragmentContainer, new PagerFragment());
        fragmentTransaction.commit();
    }
}

File: app/src/at-dinhtruong/java/asiantech/internship/summer/kotlin/recyclerview/RecyclerViewActivity.kt
package asiantech.internship.summer.kotlin.recyclerview

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import asiantech.internship.summer.R

class RecyclerViewActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_recycler_view)
        initFragment()
    }

    private fun initFragment() {
        val fragmentTransaction = supportFragmentManager.beginTransaction()
        fragmentTransaction.replace(R.id.fragmentContainer, PagerFragment.newInstance())
        fragmentTransaction.commit()
    }
}
